contracts/SortitionPool.sol

```diff
@@ -178,12 +178,12 @@ contract SortitionPool is SortitionTree, Ownable {
       selected.arrayPush(leaf);
     }
 
-    address[] memory selectedAddresses = new address[](groupSize);
+    uint32[] memory selectedIDs = new uint32[](groupSize);
 
     for (uint256 i = 0; i < groupSize; i++) {
-      selectedAddresses[i] = selected.array[i].operator();
+      selectedIDs[i] = uint32(selected.array[i].id());
```

Comment:
Can we do `uint256 -> uint32` conversion in `Leaf.id`? `Leaf.make` has `assert(_id <= type(uint32).max);` so the assumption id is safely convertible to `uint32` should stay in `Leaf` contract.

Reply:
Done https://github.com/keep-network/sortition-pools/pull/133/commits/5af2f4c2745407e4531757f85dcd1c13469e94da